framework/doc/content/source/functions/PiecewiseConstant.md

```diff
@@ -8,7 +8,7 @@ The `PiecewiseConstant` function defines the data using a set of x-y data pairs.
 of linearly interpolating between the values, however, the `PiecewiseConstant` function
 is constant when the abscissa is between the values provided by the user.  The `direction`
 parameter controls whether the function takes the value of the abscissa of the
-user-provided point to the right or left of the value at which the function is evaluated.
+user-provided point to the right, left, or centered value at which the function is evaluated.
```

Comment:
I have to confess that I didn't understand what "centered" means here.

Reply:
I'll admit I don't have a good word, `centered` made sense since it's not left, not right, but in the middle. 

Basically, when I see a list like the one in the new test `piecewise_constant_simple.i`:
```
    x = '2.5 5.0 7.0 8.5'
    y = '1   2   3   4'
```
my intuition would say, if nothing else, at 2.5 you're one, 5.0 you're 2...etc. Neither `left` nor `right` do that:
![image](https://user-images.githubusercontent.com/7014706/152017699-cfc9f95f-cbed-4366-ad99-6ea3d0695192.png)
 
And that drives me insane.

Reply:
(Disregard the sloped lines, there's no good way to plot that vertical line, This is only the function at the points)

Reply:
Okay, with right/left inclusive, I think I can reproduce the behavior:
![image](https://user-images.githubusercontent.com/7014706/152045280-babe4cc6-f5ed-4470-824e-11e872042741.png)

Reply:
This is for:
```
    x = '2.0 5.5 7.0 8.0'
    y = '1   2   3   4'
```